workspaces/src/types.rs

```diff
@@ -93,3 +96,15 @@ impl TryFrom<Vec<u8>> for CryptoHash {
         <Self as TryFrom<&[u8]>>::try_from(v.as_ref())
     }
 }
+
+impl fmt::Debug for CryptoHash {
```

Comment:
`impl std::hash::Hash` seems like a reasonable addition.

Reply:
Why would you want to hash a hash?

Reply:
So you can use it as keys to `HashMap` & `HashSet`

Reply:
Yeah, could derive it, probably worth adding

Reply:
> So you can use it as keys to `HashMap` & `HashSet`

oops had deleted my comment and updated it before I saw you responded. Sorry for my brain lapse :D